src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,76 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [Obsolete]
```

Comment:
Hi @bartonjs 
Deleting the error message was in response to Dan https://github.com/dotnet/corefx/pull/18629/commits/5beffd67c4a6f5459b8bf5b8e72d0c629c707b20#r112298863